numpy/__init__.pyi

```diff
@@ -1579,11 +1580,11 @@ class dtype(Generic[_ScalarT_co], metaclass=_DTypeMeta):
     @property
     def num(self) -> _DTypeNum: ...
     @property
-    def shape(self) -> tuple[()] | _Shape: ...
+    def shape(self) -> _AnyShape: ...
```

Comment:
out of interest, why is `shape` defined for `dtype` class?

Reply:
structured dtypes

```pycon
>>> np.dtype("(2, 3)float64").shape
(2, 3)
```

Reply:
thanks!